web/satellite/src/components/account/billing/billingTabs/Coupons.vue

```diff
@@ -4,31 +4,31 @@
 <template>
     <div class="coupon-area">
         <div class="coupon-area__top-container">
-            <h1 class="coupon-area__title">coupons</h1>
+            <h1 class="coupon-area__title">Coupon</h1>
             <VLoader v-if="isCouponFetching" />
             <div class="coupon-area__container">
                 <div
                     v-if="coupon"
                     class="coupon-area__container__existing-coupons"
                 >
-                    <div class="coupon-area__container__existing-coupons__discount-top-container active">
-                        <span class="coupon-area__container__existing-coupons__discount-top-container active">
-                            ${{ coupon.getDescription() }}
+                    <div class="coupon-area__container__existing-coupons__discount-top-container ">
+                        <span class="coupon-area__container__existing-coupons__discount-top-container__discount active-discount">
+                            {{ coupon.getDescription().slice(0, coupon.getDescription().indexOf(' ')) }}
                         </span>
                     </div>
                     <div class="coupon-area__container__existing-coupons__status-container">
-                        <span class="coupon-area__container__existing-coupons__discount-top-container active">
+                        <span class="coupon-area__container__existing-coupons__status-container__status active-status">
                             Active
                         </span>
                     </div>
                     <div class="coupon-area__container__existing-coupons__discount-black-container">
                         <span class="coupon-area__container__existing-coupons__discount-black-container__discount">
-                            ${{ coupon.getDescription() }} off
+                            {{ coupon.getDescription().slice(0, coupon.getDescription().indexOf(' ')) }} off
```

Comment:
What usually appears after the space in the coupon description?

Reply:
There is text returned that is beyond the scope of the figma.